### PR TITLE
Add multiple pages and url navigation

### PR DESCRIPTION
**Pull request by tobiola**
Added url navigation: Open to any page naming, variable naming or functionality changes

HomePage
- First Page: Display all available books
- url: "/"

BookPage
- Display and search for songs in a book
- url: "/{BookName}"

SongPage
- Display music and lyrics for a song
- url: "/{BookName}/{SongNumber}"

### CODE REVIEW

File: src/pages/BookPage.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      return (
+        <SearchView
+          searchString={searchString}
+        />
+      );
+    }
+  };
+  
+  export default BookPage;
+
```
Comment (brandonxia01) [inline src/pages/BookPage.tsx:63]: i think theres a space on the last line which is why it says no new line. Probably just need to run these through auto formatter

**Review by brandonxia01 — APPROVED** [review]
Awesome work man this is sweet!

A couple comments, and also before you merge, do you have extension `Prettier` in vscode? Can you run use that to auto format the files that you updated? With settings `use double quotes` and `line limit 120 characters`

File: src/pages/HomePage.tsx [hunk 2]
```diff
@@ -27,128 +14,20 @@ import SettingsView from "../components/SettingsView";
...
-            id="returnToHymnalButton"
-            onClick={() => setShowSettingsModal(false)}
-          >
-            Back to Hymnal
-          </IonButton>
-        </IonModal>
-
-        {GetSearchBar()}
+        {/* Redirect to book page until we add more book or make this page functional */}
+        <Redirect to="SongsOfLife" />
```
[the remark this answers is not in the document]
Comment (123asian) [reply src/pages/HomePage.tsx:25]: I think shl and sfog are good

[the remark this answers is not in the document]
Comment (tobiola) [reply src/pages/HomePage.tsx]: Should be good until we decide how we're storing books and songs. I'll just use shl for now

**Pull request merged**